yorkie/src/main/kotlin/dev/yorkie/api/ElementConverter.kt

```diff
@@ -135,13 +139,11 @@ internal fun CrdtElement.toPBJsonObject(): PBJsonElement {
 internal fun CrdtObject.toPBJsonObject(): PBJsonElement {
     val crdtObject = this
     return jSONElement {
-        PBJsonObject.newBuilder().apply {
+        jsonObject = jSONObject {
```

Comment:
@7hong13 there were a few codes that didn't use kotlin dsl version, so I've changed them.